LFPy/test/test_recextelectrode.py

```diff
@@ -595,6 +587,62 @@ def test_compare_anisotropic_lfp_methods(self):
         np.testing.assert_raises(AssertionError, np.testing.assert_array_equal,
                                  electrode_ps.LFP[0,:], electrode_sap.LFP[0,:])
 
+    def test_electrical_stimulation(self):
+        stickParams = {
+            'morphology' : os.path.join(LFPy.__path__[0], 'test', 'stick.hoc'),
+            'passive_parameters' : {'g_pas' : 1./30000, 'e_pas' : -65},
+            'passive': True,
+            'tstart' : 0,
+            'tstop' : 20,
+            'dt' : 2**-4,
+            'nsegs_method' : 'lambda_f',
+            'lambda_f' : 1000,
+
+        }
+
+        electrodeParams = {
+            'sigma' : 0.3,
+            'x' : np.array([0, 1000]),
+            'y' : np.zeros(2),
+            'z' : np.zeros(2),
+        }
+
+        def sim1():
+            electrode1 = LFPy.RecExtElectrode(**electrodeParams)
+            stick1 = LFPy.Cell(**stickParams)
+            stick1.set_pos(z=-stick1.zstart[0])
+            v1, t_ext1 = electrode1.probe.set_current_pulses(0, width1=0.1,
```

Comment:
Not so much related to LFPy, but at one point we could discuss the logic of this `MEAutility.MEA.set_current_pulses` function. Would it not  be simpler to just have this take the parameters
```
onset: float # ms, time of first pulse
amplitude: float # nA, stimulus amplitude
n_pulses: int # number of pulses
pulse_duration: float # ms, duration of pulse
pulse_interval: float, # ms, time between pulses
```
or with even fewer parameters:
```
times: ndarray # ms, times of pulses
amplitudes: const or ndarray # nA, amplitudes
duration: const # ms, length of pulses
```
For the latter the current vector could easily be constructed by a temporal convolution of a boxcar with length duration with a amplitude-hot encoding of the times on the simulation time grid

Reply:
It's a bit more complicated than that. You can also create unbalanced biphasic pulses and trains with bursts: https://github.com/alejoe91/MEAutility/blob/master/MEAutility/core.py#L556-L647

Reply:
Fine. But perhaps some of the parameters for intervals could use interval in their names, `interburst_interval` instead of `interburst`?

Reply:
Btw., I will not hold back merging because of this issue.